clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -123,6 +123,36 @@ def ConstStructAttr : CIR_Attr<"ConstStruct", "const_struct",
   let genVerifyDecl = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// IntegerAttr
+//===----------------------------------------------------------------------===//
+
+def IntAttr : CIR_Attr<"Int", "int", [TypedAttrInterface]> {
+  let summary = "An Attribute containing a integer value";
+  let description = [{
+    An integer attribute is a literal attribute that represents an integral
+    value of the specified integer type.
+  }];
+  let parameters = (ins AttributeSelfTypeParameter<"">:$type, "APInt":$value);
+  let builders = [
```

Comment:
Are both builders being used by CIRGen? If the autogenerated ones are not used, please add `let skipDefaultBuilders = 1;`.

Reply:
The `CIRGenBuilderTy` actually uses the default builders.